lib/schema.js

```diff
@@ -21,7 +21,13 @@ module.exports = {
           "properties": {
             "from": {
               "type": "string"
-            }
+            },
+            "o:tag": {
+              "type": "array"
```

Comment:
I believe that this should be 

```
"type": ["string", "array"]
```

Since we should be able to accept a single string as well as an array of string.